core/src/main/java/com/google/googlejavaformat/java/Main.java

```diff
@@ -178,8 +187,15 @@ private int formatStdin(CommandLineOptions parameters, JavaFormatterOptions opti
     }
     try {
       String output = new FormatFileCallable(parameters, input, options).call();
-      outWriter.write(output);
-      return 0;
+      if (parameters.isDryRun()) {
+        outWriter.println(STDIN_FILENAME);
```

Comment:
Let's only do this if the input was unformatted, same as for file inputs.

Reply:
Ok.